hack/postrelease/tests/test_images.py

```diff
@@ -0,0 +1,208 @@
+import json
+import subprocess
+
+import pytest
+import requests
+import variables
+
+OPERATOR_IMAGE = f"quay.io/tigera/operator:{variables.OPERATOR_VERSION}"
+
+# Architectures we expect to be present in multi-arch image manifests.
+EXPECTED_ARCHS = ["amd64", "arm64", "armv7", "ppc64le", "s390x"]
+
+# Images we expect to exist as part of a Calico release, without
+# a registry assigned.
+EXPECTED_IMAGES = [
+    "calico/node",
+    "calico/ctl",
+    "calico/apiserver",
+    "calico/typha",
+    "calico/cni",
+    "calico/kube-controllers",
+    "calico/upgrade",
+    "calico/windows",
```

Comment:
This list appears to be missing the [`node-windows`](https://quay.io/repository/calico/node-windows) and [`cni-windows`](https://quay.io/repository/calico/cni-windows) images. 
```suggestion
    "calico/windows",
    "calico/windows-upgrade",
    "calico/node-windows",
    "calico/cni-windows",
```

`windows-upgrade` and `windows` images might be obsolete now that we use host-process-container install.  @coutinhop would know for sure.

Reply:
Yes, that's correct, `calico/windows` and `calico/windows-upgrade` are obsolete, so we should only have `calico/node-windows` and `calico/cni-windows`

Reply:
```suggestion
    "calico/cni-windows",
    "calico/node-windows",
```